src/fastertransformer/core/Buffer.h

```diff
@@ -117,18 +117,38 @@ class Buffer {
 
     template<typename T>
     std::string debugDataString(size_t count) const {
+        if(where_ == MemoryType::MEMORY_GPU)
+            return "Device buffer data can NOT be dump, please use Device->clone() to convert to cpu buffer.";
```

Comment:
please add brackets for sentences under if condition even if there's only one line.

Reply:
Fixed. Thanks~